src/Administration/Resources/app/administration/src/module/sw-cms/elements/youtube-video/config/index.js

```diff
@@ -148,20 +148,31 @@ export default {
              *
              * urlLink is the link of the YouTube video from the searchbar. e.g. https://www.youtube.com/watch?v=bG57TZPYsyw
              */
-            const url = new URL(link);
-
-            switch (url.hostname) {
-                case 'www.youtu.be':
-                case 'youtu.be':
-                    return url.pathname.substring(1);
-                case 'www.youtube.com':
-                case 'youtube.com':
-                    return url.searchParams.get('v');
-                default:
-                    return link;
+
+            // If the input is already just a video ID (no protocol), return it as-is
+            if (!link.includes('://')) {
```

Comment:
I guess this is a bit of duplicate behavior: If the URL is not "complete" (for example protocol or hostname missing), it throws an error and we run into the catch you added, which does the same as this early return.